storage/byodb.md

```diff
@@ -52,3 +52,29 @@ The dynamic backends feature requires you to have one Snowflake warehouse for ea
 
 {: .image-popup}
 ![Snowflake account schema](schema.png)
+
+## BigQuery
+
+
+## BigQuery limitations
+
+Google Cloud Computing services, which include BigQuery, have introduced several limitations on their services. The following limitations are significant from the perspective of Keboola, particularly for owners of Keboola storage accounts. These limitations directly impact Keboola services and require management.
```

Comment:
> particularly for owners of Keboola storage accounts

Co to znamená?

Reply:
hmm, vlastne tu vedlejsi vetu muzeme asi vyhodit. Myslenka byla, ze to ma zajimat zejmena ty, kteri prichazi s BYODB BQ, coz jsou "vlastnici" tech backendu

Reply:
```suggestion
Google Cloud Computing services, which include BigQuery, have introduced several limitations on their services. The following limitations are significant from the perspective of Keboola, particularly for customers that are using their own BigQuery or Google Cloud Storage as storage backends. These limitations directly impact Keboola services and require management.
```